app/javascript/src/Types/libs/libdefs.js

```diff
@@ -6,3 +6,12 @@ declare var module: {
     accept(path: string, callback: () => void): void
   }
 }
+
+// TODO: remove these module declarations when not failing
+declare module 'whatwg-fetch' {
```

Comment:
Is there any issue created on their repo? We could add it in the TODO annotation

Reply:
There's no apparent issue with this package 🤷‍♂️ in fact I had no problems at all (locally) but CircleCi and @damianpm did have. So this is basically a dirty workaround to make the PR work, to be fixed in other time.